client/routes/search.js

```diff
@@ -43,6 +44,7 @@ module.exports = function (page) {
         .then(function (json) {
```

Comment:
We are using fetch-ponyfill to determine if the browser of the user support the [fetch standard](https://fetch.spec.whatwg.org/). If the fetch is supported we use the fetch from the browser otherwise we use the fetch-ponyfill implementation.